emark/message.py

```diff
@@ -193,6 +193,10 @@ def get_subject(self, **context):
             )
         return self.subject % context
 
+    def get_preheader(self):
+        """Return the email's preheader."""
```

Comment:
A little more documentation can go a long way here. Maybe don't repeat words like preheader, in case a user doesn't know the term.